src/providers/nycu.ts

```diff
@@ -25,6 +25,7 @@ export default function NYCUProvider<P extends NYCUProfile>({
     profile(profile) {
       return {
         id: profile.username,
+        name: profile.username,
```

Comment:
The username returned by NYCU auth is actually student id

Reply:
good